go.mod

```diff
@@ -17,3 +17,12 @@ require (
 	google.golang.org/genproto v0.0.0-20200526211855-cb27e3aa2013
 	google.golang.org/protobuf v1.27.1
 )
+
+require (
```

Comment:
(The dependencies in this block are already present in the module graph and already used by the `grpc` module prior to this change. `go mod tidy -go=1.17` makes them explicit in the `go.mod` file in order to support module graph pruning; see especially the second paragraph of https://go.dev/ref/mod#graph-pruning.)